telosnetwork/telos-wallet
# Adding ORE-ID Authentication support

# Fixes #463

## Description
This PR adds a new way to authenticate users using the ORE-ID service.

## Test scenarios
- (optional) Disable all your local wallets to actually see there's no interaction with local providers at all.
- go to this link
- choose login using ORE-ID
- a popup should appear
  - first time in the popup you will be asked to create your own pin to authenticate yourself
  - In normal cases, you just enter the pin you created
  - you should be authenticated, the popup should close and you should be redirected to the balances page
  - Al balances should load with no problems (even if the indexer is down)
- go to send page
- try to send any amount of TLOS.
   - It should popup a window to ask you o confirm the transaction
   - once approved the transaction go throw and the popup should close
- try to send any erc20
   - Idem
- try autologin pressing F5
   - It should Authenticate you without the need for a popup window

# Screenshots
![captura-1](https://github.com/telosnetwork/telos-wallet/assets/4420760/00036b64-b45e-4416-a4f1-da7b879fb47b)

## Checklist:
